site/fetchers/util/events.js

```diff
@@ -62,5 +62,12 @@ export function setEndDate(timeline, startDateTime, endDateTime) {
 export function eventImagePath(
   featureImageFilename) {
   const f = featureImageFilename || 'red-badger-event.jpg';
+
+  // Check if we already have full URL for the featured image
+  if (/\/\//.test(featureImageFilename)) {
+    // Check and convert http:// to https://
+    return featureImageFilename.replace(/^http:\/\//, 'https://');
```

Comment:
For the badger profile we put this check [in badger brain](https://github.com/redbadger/badger-brain/blob/master/lib/fetch/sanitize.js#L151)